sdccc/src/main/java/com/draeger/medical/sdccc/messages/mapping/MessageContent.java

```diff
@@ -75,6 +75,9 @@ public class MessageContent {
     private String uuid;
     private boolean isSOAP;
 
+    @Column(name = "sender", length = 15)
+    private String sender;
```

Comment:
Why not make the column nullable by definition? Then you could use the null value instead of "unknown".

Reply:
I never saw such a message with an "unknown" sender so far. Hence a change like that would not save us any disk space.
But thinking about it, it would make sense to use the value "UNKNOWN" or even "ERROR" in this case, to draw more attention to the fact that something went wrong.

What do you think?

Reply:
I think a nullable column serves this purpose much better, since there may be many reasons why the data is missing and the database is not the place to document why, a log message could cover the reason(s) much better.

Reply:
> I never saw such a message with an "unknown" sender so far. Hence a change like that would not save us any disk space. But thinking about it, it would make sense to use the value "UNKNOWN" or even "ERROR" in this case, to draw more attention to the fact that something went wrong.
> 
> What do you think?

I think a nullable column serves this purpose much better, since there may be many reasons why the data is missing and the database is not the place to document why, a log message could cover the reason(s) much better.

Reply:
After some discussion, we agreed that making the Column Nullable is the way to go.
I hence refactored the Code accordingly.